docs/src/pages/docs/comparison.md

```diff
@@ -8,44 +8,53 @@ title: Comparison
+| -------------------------------------------- | -------------------------------------- | -------------------------- | ------------------------------------- | ------------------------------------- |
+| Supported Query Syntax                       | Promise, REST, GraphQL                 | Promise, REST, GraphQL     | GraphQL                               | GraphQL                               |
+| Supported Query Keys                         | JSON                                   | JSON                       | GraphQL Query                         | GraphQL Query                         |
+| Query Key Change Detection                   | Deep Compare (Stable Serialization)    | Referential Equality (===) | Deep Compare (Unstable Serialization) | Deep Compare (Unstable Serialization) |
+| Query Data Memoization Level                 | Query + Structural Sharing             | Query                      | Query + Entity + Structural Sharing   | Query + Entity + Structural Sharing   |
+| Bundle Size                                  | [![][bp-react-query]][bpl-react-query] | [![][bp-swr]][bpl-swr]     | [![][bp-apollo]][bpl-apollo]          | [![][bp-react-relay]][bpl-react-relay]|
+| Queries                                      | ✅                                     | ✅                         | ✅                                    | ✅                                    |
+| Caching                                      | ✅                                     | ✅                         | ✅                                    | ✅                                    |
+| Devtools                                     | ✅                                     | 🟡                         | ✅                                    | ✅                                    |
+| Polling/Intervals                            | ✅                                     | ✅                         | ✅                                    | ✅                                    |
+| Parallel Queries                             | ✅                                     | ✅                         | ✅                                    | ✅                                    |
+| Dependent Queries                            | ✅                                     | ✅                         | ✅                                    | ✅                                    |
+| Paginated Queries                            | ✅                                     | ✅                         | ✅                                    | ✅                                    |
+| Infinite Queries                             | ✅                                     | ✅                         | ✅                                    | ✅                                    |
+| Lagged / "Lazy" Queries<sup>1</sup>          | ✅                                     | 🛑                         | 🛑                                    | ✅                                    |
```

Comment:
In the beta version of the comparison, this has been changed to `Lagged Query Data`, which has nothing to do with what the world understands as "lazy" querying. This essentially means that the previous query results can be lagged or kept in the query as new ones are fetched. Imagine paging through some results. Normally, with a new page, you would see a hard loading state since the query and/or variables have changed. This allows you to continue to display the latest successful data while a completely. new query is fetched in the background to replace it.

Can you verify this?

Reply:
I think Relay does this with the "store-and-network" fetchPolicy.
```
fetchPolicy: Optional. Determines if cached data should be used, and when to send a network request based on the cached data that is currently available in the Relay store (for more details, see our Fetch Policies and Garbage Collection guides):
"store-or-network": (default) will reuse locally cached data and will only send a network request if any data for the query is missing. If the query is fully cached, a network request will not be made.
"store-and-network": will reuse locally cached data and will always send a network request, regardless of whether any data was missing from the local cache or not.
"network-only": will not reuse locally cached data, and will always send a network request to fetch the query, ignoring any data that might be locally cached in Relay.
```

Reply:
That doesn't sound like what I described. A better test question might be:

If you change the query or variables to an *uncached or new* query permutation, can you configure it to continue to display the previous query's data while the new/uncached one fetches?

Reply:
I think it's not possible with another lib, maybe they are waiting for useTransition to be available

Reply:
Only in the experimental version with the new concurrent api's like usePending (??) but there's no icon for that and we're waiting for concurrent mode like forever :-D